cfgov/unprocessed/css/cf-theme-overrides.less

```diff
@@ -224,7 +224,7 @@
 @category-slug-hover:           @link-text-hover;
 
 // .header-slug
-@header-slug-thin-border:       @gray-20;
+@header-slug-thin-border:       @gray-50;
```

Comment:
I talked with @ajbush about this. Previously the borders didn't match, so we changed them to gray-50.